CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -0,0 +1,100 @@
+- Release Antrea Helm chart archive for each Antrea release. ([#3935](https://github.com/antrea-io/antrea/pull/3935) [#3952](https://github.com/antrea-io/antrea/pull/3952), [@antoninbas] [@yanjunz97])
+  * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.8/docs/helm.md) for Helm installation method. ([#3989](https://github.com/antrea-io/antrea/pull/3989), [@antoninbas])
+- Support TopologyAwareHints in AntreaProxy. ([#3515](https://github.com/antrea-io/antrea/pull/3515), [@hongliangl])
+- Add support for encap mode to the Multicast feature. ([#3947](https://github.com/antrea-io/antrea/pull/3947), [@wenyingd])
+- Support configurable tunnel port for encap mode. ([#4065](https://github.com/antrea-io/antrea/pull/4065), [@Jexf])
+- Add Status field to the IPPool CRD: it is used to report usage information for the pool (total number of IPs in the pool and number of IPs that are currently assigned). ([#3072](https://github.com/antrea-io/antrea/pull/3072) [#4088](https://github.com/antrea-io/antrea/pull/4088), [@ksamoray] [@tnqn])
+- Support updating configuration at runtime for flow-aggregator via antctl or updating ConfigMap. ([#3642](https://github.com/antrea-io/antrea/pull/3642), [@yuntanghsu])
+- Add antctl commands to set up and delete Multi-cluster ClusterSet. ([#3992](https://github.com/antrea-io/antrea/pull/3992), [@hjiajing])
+- Add [documentation](https://github.com/antrea-io/antrea/blob/release-1.8/docs/multicluster/antctl.md) to set up Multi-cluster ClusterSet with antctl. ([#4096](https://github.com/antrea-io/antrea/pull/4096), [@jianjuns])
+
+### Changed
+
+- Antrea now uses OpenFlow 1.5 to program OVS. ([#3770](https://github.com/antrea-io/antrea/pull/3770), [@wenyingd] [@ashish-varma])
+- Rename Windows script Start.ps1 to Start-AntreaAgent.ps1, and rename Stop.ps1 to Stop-AntreaAgent.ps1. ([#3904](https://github.com/antrea-io/antrea/pull/3904), [@wenyingd])
+- Unify NodePortLocal behavior across Linux and Windows. Linux agents now support allocating different nodeports for different protocols when the podports are the same. ([#3936](https://github.com/antrea-io/antrea/pull/3936), [@XinShuYang])
+- Antrea IPAM now uses the name of the uplink interface to name the host internal port, the uplink interface will be renamed with a `~` suffix, e.g. `eth0~`. ([#3938](https://github.com/antrea-io/antrea/pull/3938), [@gran-vmv])
+- Send Neighbor Advertisement messages after creating Pods in IPv6 cluster. ([#3998](https://github.com/antrea-io/antrea/pull/3998), [@gran-vmv])
+- Add columns of kubectl outputs for Multi-cluster types. ([#3923](https://github.com/antrea-io/antrea/pull/3923), [@jianjuns])
+- Add IP version and prefix length validation to IPPool CRD manifest. ([#3897](https://github.com/antrea-io/antrea/pull/3897), [@jianjuns])
```

Comment:
I feel this one is too trivial to add (and it does not really change the API behavior but just the implementation).

Reply:
removed